content/zh/docs/toolbox/metering-and-billing/enable-billing.md

```diff
@@ -5,3 +5,39 @@ description: "启用 KubeSphere 中的计费功能，查看一段时间内您资
 linkTitle: "启用计费"
 weight: 15420
 ---
+
+## 开启计费功能
+
+默认计费功能是关闭的，可以通过以下步骤开启计费功能：
+
+1. 在 configmap `ks-metering-config` 中指定不同资源类型的价格（初始为-1）
+
+```shell
+# kubectl edit cm ks-metering-config -n kubesphere-system
+
+# kubectl get cm ks-metering-config -n kubesphere-system -oyaml
+apiVersion: v1
+data:
+  ks-metering.yaml: |
+    retentionDay: 30d
+    billing:
+      priceInfo:
+        currencyUnit: "USD"
+        cpuPerCorePerHour: 1.5
+        memPerGigabytesPerHour: 5
+        ingressNetworkTrafficPerMegabytesPerHour: 1
+        egressNetworkTrafficPerMegabytesPerHour: 1
+        pvcPerGigabytesPerHour: 2.1
+kind: ConfigMap
+metadata:
+  name: ks-metering-config
+  namespace: kubesphere-system
+```
+
+> 上面定义了包含货币单位在内的价格信息：货币目前支持人民币 (CNY) 和美元 (USD) ，举例来说，上面配置的含义是 CPU 每 core 每小时 1.5 美元，内存每 GB 每小时 5 美元， ingress 网络流量每 MB 每小时 1 美元， egress 网络流量每 MB 每小时 1 美元， PVC 每 GB 每小时 2.1 美元。
```

Comment:
What happened if a user gives an unsupported currency?

Reply:
Yeah, currently there's no currency unit check. I will discuss this part with @harrisonliu5

Reply:
What I want to say is that we can tell users what happened if they provide an unsupported currency. For example:

* Use a default currency
* Or show tips to let users know they just provide an unsupported currency

Reply:
If user set other currency unit, console will display "CNY".

Reply:
Got it.